test/fail_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+NC='\033[0m'
+CYAN='\033[0;36m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+
+INPUT_FILES="compiler/fail/*.ods"
+TMP_FILE="temp.py"
+printf "${CYAN}Running compiler tests...\n${NC}"
+
+for input_file in $INPUT_FILES; do
+    expected_error_file=${input_file/.ods/.out}
+    exp_err=$(tr -d "[:space:]" < $expected_error_file)
```

Comment:
I was getting errors with this even though the spacing looked correct. It could mean failing tests even if they are meant to be passing because of an extra space here or there. I'm happy to put it back in but I think it would save us some grief by taking it out.